service/eventer/github/client.go

```diff
@@ -25,10 +20,10 @@ const (
 	// See: https://en.wikipedia.org/wiki/HTTP_ETag.
 	etagHeader = "Etag"
 
-	// LatestDeploymentUrlFormat is the format string used to compute the Github
+	// DeploymentUrlFormat is the format string used to compute the Github
 	// API URL used to fetch the latest deployment event for a specific
 	// environment.
-	LatestDeploymentUrlFormat = "https://api.github.com/repos/%s/%s/deployments?environment=%s"
+	DeploymentUrlFormat = "https://api.github.com/repos/%s/%s/deployments?environment=%s"
```

Comment:
I noticed the URLs are the same and we do not need to do the filtering by environment by ourselves. Github does it for us.